scripts/gen_eval_files.lua

```diff
@@ -211,6 +211,39 @@ local function get_api_meta()
   return ret
 end
 
+---@param text string
```

Comment:
Both styles are allowed.
See the updated `dev-lua-doc`:

```
Docstring format:
- Use LuaCATS annotations: https://luals.github.io/wiki/annotations/
- Limited markdown is supported.
  - List-items start with `-` (useful to nest or "indent")
- Use ``` for code samples.
  Code samples can be annotated as `vim` or `lua`
- Use `@nodoc` to prevent documentation generation.
- Files which has `@meta` are only used for typing and documentation.
```